apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/internal/SchemaUploader.kt

```diff
@@ -0,0 +1,43 @@
+package com.apollographql.apollo.gradle.internal
+
+import com.apollographql.apollo.compiler.fromJson
+import com.apollographql.apollo.gradle.internal.SchemaDownloader.cast
+import java.net.InetAddress
+import java.nio.charset.StandardCharsets
+import java.security.MessageDigest
+import java.util.UUID
+
+object SchemaUploader {
+  fun uploadSchema(key: String, graph: String, variant: String, sdl: String) {
```

Comment:
I think `key`, `graph`, `variant`, and `sdl` should all be required String types here

Reply:
I think that's the case already?

Reply:
oh heh, I was reading this as GraphQL, not Kotlin 😂  my bad!